README.adoc

```diff
@@ -19,6 +19,9 @@
 
 = Eclipse Microprofile Metrics
 
+[red]*NOTE: The 2.1 branch is discontinued and should not be used, there will be no more 2.1.x update releases.
+2.2 release was put out to fix issues in 2.1* 
```

Comment:
This makes it sound like there may have been lots of problems in 2.1.  might be better to say "2.2 release was put out to fix an API incompatibility between 2.1 and 2.0"?

Reply:
Hm, I guess you're right. For some reason I wanted to avoid stating explicitly what it was, but I guess it will be best